tasks/utils.js

```diff
@@ -120,10 +120,11 @@ function copyFile(srcPath, dstPath) {
     if (isFile(srcPath)) fs.copyFileSync(srcPath, dstPath);
 }
 
-function copyFileToDir(srcPath, dstDir) {
-    var lastSlash = srcPath.lastIndexOf("/");
+function copyFileToDir(srcPath, dstDir, useSlash = true) {
+    var slash = useSlash ? "/" : "\\"
```

Comment:
does this actually matter?
i thought / worked fine on windows, except for only the cmd shell